build/Dockerfile

```diff
@@ -63,6 +63,18 @@ RUN set -x \
 	&& mkdir -p $(dirname $CLUSTER_PATH) \
 	&& chown $STEAM_USER:$STEAM_USER $(dirname $CLUSTER_PATH)
 
+ARG MODS
+ENV INSTALLED_MODS ${MODS}
```

Comment:
Can we name the `ENV` variable also `MODS` instead of `INSTALLED_MODS`?

Reply:
I prefer to keep `INSTALLED_MODS` as a _private_ `ENV`. So a different name is more _secure_.

Reply:
I'm not sure I can follow. How does it differ by using `MODS` only?

Reply:
Hmmm... I was thinking about this, and right now, split it have no sense. My idea in this PR was to remove the `--update=mods` option. But then I re-thought it. This change was made with that in mind.